kasa/tests/test_device_factory.py

```diff
@@ -21,8 +21,9 @@
 
 
 def _get_connection_type_device_class(the_fixture_data):
-    if "discovery_result" in the_fixture_data:
-        discovery_info = {"result": the_fixture_data["discovery_result"]}
+    if "result" in the_fixture_data:
+        # discovery_info = {"result": the_fixture_data["discovery_result"]}
```

Comment:
Remove or document why this should be kept as a comment.

Reply:
Removed